README.md

```diff
@@ -440,151 +517,230 @@ Before setting up the following Survey application, dependencies given below sho
+            ```
+      -  **Windows**
+
+         1. Run Migrations:
+
+            ```
+            cd user\src && npx sequelize-cli db:migrate && cd ..\.. &&
+            cd notification\src && npx sequelize-cli db:migrate && cd ..\..
+            ```
 
 7.  **Enabling Citus And Setting Distribution Columns (Optional)**
 
       To boost performance and scalability, users can opt to enable the Citus extension. This transforms PostgreSQL into a distributed database, spreading data across multiple nodes to handle large datasets more efficiently as demand grows.
 
       > NOTE: Currently only available for Linux based operation systems.
 
-  
 
       1. Download user `distributionColumns.sql` file.
-   
-         ```
-         curl -o ./user/distributionColumns.sql -JL https://raw.githubusercontent.com/ELEVATE-Project/samiksha-service/refs/heads/main/documentation/1.0.0/user/distributionColumns.sql
-         ```
+
+         -  **Linux/Ubuntu/MacOS**
+            ```
+            curl -o ./user/distributionColumns.sql -JL https://raw.githubusercontent.com/ELEVATE-Project/samiksha-service/refs/heads/main/documentation/1.0.0/user/distributionColumns.sql
+            ```
+      
+         -  **Windows**
+            ```
+            curl -o .\user\distributionColumns.sql -JL https://raw.githubusercontent.com/ELEVATE-Project/samiksha-service/refs/heads/main/documentation/1.0.0/user/distributionColumns.sql
+            ```
+
       2. Set up the `citus_setup` file by following the steps given below.
-   
-         -  **Ubuntu/Linux**
-   
+
+         - **Ubuntu/Linux**
+
             1. Download the `citus_setup.sh` file:
-   
                ```
                curl -OJL https://raw.githubusercontent.com/ELEVATE-Project/samiksha-service/refs/heads/main/documentation/1.0.0/native/scripts/linux/citus_setup.sh
                ```
+
             2. Make the setup file executable by running the following command:
-   
                ```
                chmod +x citus_setup.sh
                ```
+
             3. Enable Citus and set distribution columns for `user` database by running the `citus_setup.sh`with the following arguments.
-   
                ```
                ./citus_setup.sh user postgres://postgres:postgres@localhost:9700/users
                ```
+         
+         - **Windows**
+
+            1. Download the `citus_setup.bat` file:
+               ```
+               curl -OJL https://raw.githubusercontent.com/ELEVATE-Project/samiksha-service/refs/heads/main/documentation/1.0.0/native/scripts/windows/citus_setup.sh
+               ```
+            
+            2. Enable Citus and set distribution columns for `user` database by running the `citus_setup.bat`with the following arguments.
+               ```
+               citus_setup.bat user postgres://postgres:postgres@localhost:9700/users
+               ```
+
 8.  **Insert Initial Data**
 
      - **Ubuntu/Linux/Mac/Windows**
```

Comment:
@MallanagoudaB commands mentioned under Step 8 will not work for windows setup. Please check.

Reply:
this will work for all OS @Prajwal17Tunerlabs